Documentation/general/cli-ux-guidelines.md

```diff
@@ -0,0 +1,155 @@
+.NET Core Command-Line Tools UX Guidelines
+-------------------------------------------
+
+This document will cover the CLI UX guidelines that anyone adding or suggesting
+a new command should take into consideration. It does not deal with
+implementation details. The main goal of the document is to provide guidelines
+for new authors that wish to make pull requests and add new functionalities and
+commands to the .NET Core command line tools (CLI).
+
+## Naming the commands
+
+In the .NET Core CLI, commands should be **verbs**. This rule was adopted

```

Comment:
> In the .NET Core CLI

I don't think you need to contextualize it. You may need to define (in an earlier section) what commands are. Outside that, just say "Commands should be **verbs**."

I suggest sticking with **guidelines** and not adopting *rules**.